apps/backend/src/utils/auth.ts

```diff
@@ -20,6 +20,17 @@ export const auth = betterAuth({
   database: prismaAdapter(prismaClient, {
     provider: 'postgresql',
   }),
+  socialProviders: {
+    google: {
+      clientId: process.env.GOOGLE_CLIENT_ID as string, //!expecting client id from env file
+      clientSecret: process.env.GOOGLE_CLIENT_SECRET as string, //!expecting client secret from env file
+    },
+    github: {
+      clientId: process.env.GITHUB_CLIENT_ID as string, //!expecting client id from env file
+      clientSecret: process.env.GITHUB_CLIENT_SECRET as string, //!expecting client secret from env file
+    },
+  },
+
   emailAndPassword: {
```

Comment:
in this file on Line 90, we're also tracking the provider the user used upon Signup:

`await vemetric.trackEvent('Signup', { userIdentifier: user.id, eventData: { provider: 'email' } });`

here we should pass the correct provider used (email, GitHub, Google) .. there we need to get it from the account table from better-auth .. probably need to create a new `auth-account.ts` file in the `database` package where we query it via the userId .. but maybe there's also a better solution provided by better-auth